### PR TITLE
Create program areas page

### LINKED ISSUE #833: Create Program Areas page
### Overview

We need to create a program area page so that people can browse by topic area.

### Action Items

Product

- [x] To define the program areas required.
- [x] Write a header text.

Design

- [x] Use communities of practice as base the template replacing the team members with project links based on project display from events page location view.

Development

- [ ] Reach out to @daniellex0 (@ daniellenedivi on slack) to go over which reusable components to use and how
- [x] Reference new [template for developing a new page](https://github.com/hackforla/website/wiki/How-to-create-a-page-on-Hack-for-LA-website) on the HfLA site (via @martham0 )
- [x] Develop the program area page based on the Figma file
   - [x] Create .md file(s) for anyone to add or remove projects / edit program area descriptions over time

UX Writing

- [ ] Finish writing copy for the page

### Resources

[Figma of program area page](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=4452%3A21046) (final design in pink rectangle)

[Template / wiki for developing a new page on the HfLA site](https://github.com/hackforla/website/wiki/How-to-create-a-page-on-Hack-for-LA-website)

[Doc with program areas](https://docs.google.com/document/d/1YPcbq9OsE5i1KKXsYExxuKZmnAuWLItvQfukaKFrujo/edit)

### PR DESCRIPTION
**Pull request by akibrhast**
### Create Program Areas Page As following the issue instructions on #833.

Hey folks who are going to be reviewing this PR, 
I got the framework for the Program Areas Page setup and running. What's left is drilling down on the designs. Since this page is not linked and under active development. I would like to merge this issue such that another developer can continue working of off from the current state. I believe after merging this, we could possibly open up two small separate issues that goes along the line of the following title

1. Fix design and alignment of project mini-card such that it matches figma specs
    - labels: medium issue,ui,front-end
2. Add join a project section right above the footer following figma specs
    - labels: medium issue,ui,front-end

<details>
<summary>Current State Of Program Areas Page</summary>

image
</details>

### CODE REVIEW

File: pages/program-areas.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    </div>
+    <div class="program-areas-header__hero-image">
+        {% include svg/program-areas-hero.svg %}
+    </div>
+</section> 
+<section class="content-section content--program-areas">
+    {% assign sorted_program_areas = site.data.internal.program-areas | sort %}
+    {% for program_areas in sorted_program_areas %}
+        {% if program_areas[1].size > 0%}
+            <div class="page-card-base card-primary page-card-lg page-card-container">
```
Comment (averdin2) [inline pages/program-areas.html:22]: @akibrhast Based off of the Figma, it seems that where it has class "page-card-lg" on line 22, it should be "page-card-xl". I know that you are looking to make further changes to this page into smaller issues, but is there any reason why this shouldn't be "page-card-xl" now? Other than this, everything looks good to me.

**Review by averdin2 — APPROVED** [review]
Other than the mentioned comment about "page-card-lg" class, everything looks good to me.

File: pages/program-areas.html (hunk 1 above)
Comment (akibrhast) [reply pages/program-areas.html:22]: @averdin2 Thanks for your in depth review.

In regards to your question. If the figma says so then yes, it should be changed to page-card-xl on this issue before it is merged.

Unfortunately I was not able to locate in the figma designs where it says page-card-xl(i am not super used to navigating figma). Could you please provide a screenshot or some kind of demo as to where I should be looking at?

**Comment by akibrhast** [comment]
- [x] Reach out to @daniellex0 (@ daniellenedivi on slack) to go over which reusable components to use and how

Reaching out to  @daniellex0 . Could you please give your comments and feedback ?

File: pages/program-areas.html (hunk 1 above)
Comment (averdin2) [reply pages/program-areas.html:22]: @akibrhast I don't believe it says that in the Figma. I am more just going off of how the design looks vs how the website looks with the current implementation.

**Comment by daniellex0** [comment]
Hey y'all, sorry I can only get to this later tonight because I have to prepare for UX community of practice meeting tonight. But regarding the reusable classes- Candace, Qiqi, and I are currently working on documenting these better for developers. Is it possible to hold off on this until I can get to it tonight? Also re: the page cards, if the width is 896px it should indeed be page-card-lg . And you can check the [Communities of Practice](https://www.hackforla.org/communities-of-practice) page for a similar example

File: pages/program-areas.html (hunk 1 above)
Comment (akibrhast) [reply pages/program-areas.html:22]: @averdin2 Based on daniels comment below `Also re: the page cards, if the width is 896px it should indeed be page-card-lg . And you can check the Communities of Practice page for a similar example` as well as this screenshot. I believe page-card-lg is correct in this instance. As such I am going to mark this conversation as resolved. Please feel free to correct me tough, if you feel this is still unresolved.

image

**Review by jbubar — APPROVED** [review]

**Pull request merged**